lib/User.js

```diff
@@ -93,7 +93,7 @@ class User extends Requestable {
 
    /**
     * List users followed by another user
```

Comment:
Could you update this comment to match? Something like:
Lists the people who the authenticated user follows.

Reply:
Sure!

Reply:
😄